src/services/user.js

```diff
@@ -0,0 +1,39 @@
+
```

Comment:
Can we get rid of this blank line? The opening comment should be at the very top of the file.